### PR TITLE
DT-1642: ✨ Add create group

### PR DESCRIPTION
**Pull request by petergphillips**

### CODE REVIEW

File: backend/controllers/createGroup.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+const { validateCreateGroup } = require('./groupValidation')
+const { trimObjValues } = require('../utils')
+
+const createGroupFactory = (createGroup, manageGroupUrl) => {
```
Comment (petergphillips) [inline backend/controllers/createGroup.js:4]: copied from createChildGroup and removed parent stuff

File: views/createChildGroup.njk [hunk 2]
```diff
@@ -4,11 +4,6 @@
 {% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
 {% from "govuk/components/input/macro.njk" import govukInput %}
 
-{% block header %}
```
Comment (petergphillips) [inline views/createChildGroup.njk:7]: autocomplete not used on this page

File: views/createGroup.njk [hunk 3]
```diff
@@ -0,0 +1,72 @@
+{% extends "./partials/layout.njk" %}
+{% from "govuk/components/select/macro.njk" import govukSelect %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk/components/input/macro.njk" import govukInput %}
+
+{% set title =  "Create group" %}
```
Comment (petergphillips) [inline views/createGroup.njk:7]: copied from createChildGroup.njk

**Review by SimonMitchellMOJ — APPROVED** [review]
LGTM

**Pull request merged**